src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -81,30 +82,59 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
     apiOperations
   )
 
-  import kantan.csv.generic._
-  import tech.cryptonomic.conseil.util.ConfigUtil.Csv._
+  implicit val (tokens, tns) = initAnyCsvConfig()
 
-  /** Inits registered tokens at startup */
-  implicit val tokenContracts: TokenContracts = {
+  /* Reads csv resources to initialize db tables and smart contracts objects */
+  def initAnyCsvConfig(): (TokenContracts, TNSContracts) = {
+    import kantan.csv.generic._
+    import tech.cryptonomic.conseil.util.ConfigUtil.Csv._
 
-    val futureTokenContracts =
-      TezosDb.initTableFromCsv(db, Tables.RegisteredTokens, tezosConf.network, separator = '|').map {
-        case (tokenRows, _) =>
-          TokenContracts.fromTokens(
-            tokenRows.map {
-              case Tables.RegisteredTokensRow(_, tokenName, standard, accountId) =>
-                ContractId(accountId) -> standard
-            }
-          )
+    val tokenContracts: TokenContracts = {
+
+      val futureTokenContracts =
```

Comment:
Minor gripe..can we rename this to something like `tokenContractsFuture`?

Reply:
done